src/main/java/mate/academy/spring/validation/password/PasswordValidator.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.spring.validation.password;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class PasswordValidator implements ConstraintValidator<Password, String> {
+    private static final Pattern PATTERN = Pattern.compile("^\\+380\\d{9}$");
+
+    @Override
+    public boolean isValid(String password, ConstraintValidatorContext constraintValidatorContext) {
+        return password != null && PATTERN.matcher(password).matches();
+    }
+}
```

Comment:
you don't need to use a regex here, you need to check that the password equals the repeat password

Reply:
Yes, you're right, I've made something strange here